vignettes/basic_data_wrangling.Rmd

```diff
@@ -0,0 +1,518 @@
+---
+title: "Basic Data Wrangling"
+output: 
+  rmarkdown::html_vignette:
+    toc: true
+    fig_width: 10.08
+    fig_height: 6
+vignette: >
+  \usepackage[utf8]{inputenc}
+  %\VignetteIndexEntry{Basic Data Wrangling}
+  %\VignetteEngine{knitr::rmarkdown}
+---
+
+```{r message=FALSE, warning=FALSE, include=FALSE}
+library(knitr)
+options(knitr.kable.NA = "")
+knitr::opts_chunk$set(
+  eval = FALSE,
+  message = FALSE,
+  warning = FALSE,
+  dpi = 300
+)
+
+pkgs <- c(
+  "datawizard",
+  "tidyr",
+  "dplyr",
+  "see",
+  "ggplot2",
+  "parameters",
+  "lme4"
+)
+
+if (!all(sapply(pkgs, requireNamespace, quietly = TRUE))) {
+  knitr::opts_chunk$set(eval = FALSE)
+}
+```
+
+
+```{css, echo=FALSE, eval = TRUE}
+.datawizard, .datawizard > .sourceCode {
+  background-color: #e6e6ff;
+}
+.tidyverse, .tidyverse > .sourceCode {
+  background-color: #d9f2e5;
+}
+```
+
+`datawizard` is a package whose aim is to make basic data wrangling easier than 
+with base R. Although it is inspired by the tidyverse combo `dplyr` + `tidyr`, 
```

Comment:
We should refer to packages as `{dplyr}`, and not as `dplyr`, to distinguish a package name from an object name.

Reply:
done